assets/js/modules/adsense/datastore/adblocker.test.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * modules/adsense data store: adblocker tests.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { STORE_NAME } from './constants';
+import {
+	createTestRegistry,
+	subscribeUntil,
+	unsubscribeFromAll,
+} from '../../../../../tests/js/utils';
+
+describe( 'modules/adsense adblocker', () => {
+	let registry;
+	let select;
+	let dispatch;
+
+	beforeEach( () => {
+		registry = createTestRegistry();
+		select = registry.select( STORE_NAME );
+		dispatch = registry.dispatch( STORE_NAME );
+	} );
+
+	afterEach( () => {
+		unsubscribeFromAll( registry );
+	} );
+
+	describe( 'actions', () => {
+		describe( 'receiveIsAdBlockerActive', () => {
+			it( 'requires the isAdBlockerActive param', () => {
+				expect( () => {
+					dispatch.receiveIsAdBlockerActive();
+				} ).toThrow( 'isAdBlockerActive is required.' );
+			} );
+
+			it( 'receives and sets isAdBlockerActive', async () => {
+				await dispatch.receiveIsAdBlockerActive( true );
```

Comment:
Does this test need to be async? I don't think it does (not a big deal, but just saying).